platform/accessibility/vsp-accessibility-rules-of-engagement.md

```diff
@@ -0,0 +1,93 @@
+
+   * Reach out to your accessibility specialist(s) if you have questions after the product kickoff meeting. The sooner your team is comfortable with these accessibility guidelines, the more successful your project will be.
+
+2. **Design Intent Collaboration** — *Integrated with [VSP design intent check-in](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/design/working-with-platform-design-team.md#design-intent-check-in)*<br/><br/>Tips for designing accessible solutions:
+
+   * Involve the accessibility specialist in discussions about needs, priorities, concepts, research planning, and direction.
+
+   * Include the accessibility specialist in formal and informal reviews to ensure inclusive design is woven through the efforts. Accessibility specialists can help identify challenges with user flows, color contrast, and color blindness at this stage of the software development lifecycle (SDLC).
+
+   * Consider the following elements while designing user experiences:
+     * [Performance budgets](https://addyosmani.com/blog/performance-budgets/)
+     * Mobile-first design 
+     * Mobile-only interactions
+     * Touch interactions
+     * Spacing of interactive items – for example, is the spacing sufficient to ensure people don't accidentally select something other than what they meant to?
+     * Laptop/desktop design
+     * Semantic heading order
+     * Cognitive impact – in the words of [Steve Krug](http://sensible.com/), *don't make me think!* Design should be clear, not overwhelming.
+
+3. **Usability Testing Prep** — *Integrated with [VSP pre-usability design review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/design/working-with-platform-design-team.md#design-reviews-1)*<br/><br/>The accessibility specialist will provide more specific feedback, including:
+
+   * Relevant design system components and patterns
+
+   * Feedback on recruiting diverse research participants
+
+   * Additional testing scenarios
+
+4. **Staging Review** — *Integrated with [VSP design QA review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/design/working-with-platform-design-team.md#design-reviews-1)*
+
+   * The accessibility specialist will review the following items:
+     * A completed test plan. This test plan could be stored in [TestRail](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc) **or** [Github](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/accessibility-test-cases-template.md). **Incomplete test plans can delay the staging accessibility review.**
+     * Automated end-to-end (e2e) tests for client-side applications.
```

Comment:
I would love to have something in the Staging Review request template where the requestor certifies that they have the e2e axe checks in their code, AND that they have completed the following, and addressed issues:
- [ ] run axe browser scan (which includes color contrast checks)
- [ ] run axe-coconut browser scan (which includes color contrast checks)
- [ ] zoomed to 400% with the viewport width of 1280px
- [ ]  read the project work in one screen reader, and indicate which one